apps/etterlatte-statistikk/src/main/kotlin/statistikk/service/StatistikkService.kt

```diff
@@ -256,6 +259,44 @@ class StatistikkService(
         )
     }
 
+    private fun klageTilSakRad(
+        statistikkKlage: StatistikkKlage,
+        tekniskTid: LocalDateTime,
+        hendelse: KlageHendelseType,
+    ) = SakRad(
+        id = -1,
+        referanseId = statistikkKlage.klage.id,
+        sakId = statistikkKlage.klage.sak.id,
+        mottattTidspunkt = statistikkKlage.klage.opprettet,
+        registrertTidspunkt = statistikkKlage.tidspunkt,
+        type = hendelse.name,
+        status = statistikkKlage.klage.status.name,
+        resultat = statistikkKlage.klage.kabalResultat?.name,
+        saksbehandler = statistikkKlage.saksbehandler,
+        ansvarligEnhet = statistikkKlage.klage.sak.enhet,
+        ansvarligBeslutter = statistikkKlage.klage.formkrav?.saksbehandler?.ident,
+        aktorId = statistikkKlage.klage.sak.ident,
+        tekniskTid = tekniskTid.toTidspunkt(),
+        sakYtelse = statistikkKlage.klage.sak.sakType.name,
+        avdoedeForeldre = null,
+        sakYtelsesgruppe = null,
+        revurderingAarsak = null,
+        vedtakLoependeFom = null,
+        vedtakTidspunkt = null,
+        beregning = null,
+        avkorting = null,
+        kilde = Vedtaksloesning.GJENNY,
```

Comment:
Denne kan være 
```suggestion
        kilde = "BRUKER".takeIf(statistikkKlage.klage.innkommendeDokument?.innsender != null) ?: VedtaksLoesning.GJENNY 
```
Med tillegg av bruker eller whatever i typen for denne